BigQuery/src/Connection/Rest.php

```diff
@@ -31,7 +31,8 @@
  * Implementation of the
  * [Google BigQuery JSON API](https://cloud.google.com/bigquery/docs/reference/v2/).
  *
- * @internal
+ * @deprecated This class is no longer supported and will be removed in a future
```

Comment:
I think we should keep these as `@internal` (I didn't realize these were already marked internal - that's helpful)

Reply:
The `Rest`, `Grpc` classes and `ConnectionInterfaces` are all marked `@deprecated`. Should I make an exception here or for consistency keep `@deprecated` here as well.